tools/benchmark/cmd/watch.go

```diff
@@ -169,7 +165,9 @@ func watchFunc(cmd *cobra.Command, args []string) {
 		close(putreqc)
 	}()
 
-	<-recvCompletedNotifier
+	for range streams {
+		<-recvCompletedNotifier
```

Comment:
Also change `recvWatchChan` so `recvCompletedNotifier <- struct{}{}` happens outside the `range wch` loop?